firedrake/slate/slac/compiler.py

```diff
@@ -43,7 +43,12 @@
 __all__ = ['compile_expression']
 
 
-PETSC_DIR = get_petsc_dir()
+# Can't I assume that PETSC_ARCH is set ? or should I patch pyOP2 to exit when ARCH is not set ?
```

Comment:
No,  if PETSc was installed with a prefix install, then `PETSC_ARCH` is not needed. So you should set `PETSC_ARCH` to "" in that case.

Reply:
Ok so in that case, where would the conf files be ? $PREFIX/petsc/conf ?
How do I retrieve the prefix ?

Reply:
`get_petsc_dir()` should provide it.